spec/models/beta/search/open_search_result_spec.rb

```diff
@@ -266,6 +266,15 @@
     it { expect { result.redirect! }.to change(result, :redirect?).from(nil).to(true) }
   end
 
+  describe '#intercept_message' do
+    subject(:result) { build(:search_result) }
+
+    it 'will match the object in the yaml file' do
+      searched_term = result.search_query_parser_result.corrected_search_query
+      expect(searched_term).to eq((I18n.t "#{searched_term}.title"))
+    end
+  end
+
```

Comment:
should we add the context where searched_term is not present?
Is it a common scenario, which we can handle returning nil
or is it a rare scenario, which should be handled by exception? 🤔

Reply:
If the searched term is not present then we currently get the message and title that it does not exist in the `yaml` file. But we can tweak it as you've mentioned and return nil instead, if it doesn't exist over there. I'll modify this edge case and ping you shortly :)